src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Net46.cs

```diff
@@ -0,0 +1,1193 @@
+                return _serverCertificateCustomValidationCallback;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _serverCertificateCustomValidationCallback = value;
+            }
+        }
+
+        public SslProtocols SslProtocols
+        {
+            // TODO: Issue #11100.
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        public virtual bool SupportsAutomaticDecompression
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        public virtual bool SupportsProxy
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        public virtual bool SupportsRedirectConfiguration
+        {
+            get
+            {
+                return true;
+            }
+        }
+
+        public bool UseCookies
+        {
+            get
+            {
+                return _useCookies;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _useCookies = value;
+            }
+        }
+
+        public bool UseDefaultCredentials
+        {
+            get
+            {
+                return _useDefaultCredentials;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _useDefaultCredentials = value;
+            }
+        }
+
+        public bool UseProxy
+        {
+            get
+            {
+                return _useProxy;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _useProxy = value;
+            }
+        }
+
+        #endregion Properties
+
+        #region De/Constructors
+
+        public HttpClientHandler()
+        {
+            _startRequest = StartRequest;
+            _getRequestStreamCallback = GetRequestStreamCallback;
+            _getResponseCallback = GetResponseCallback;
+
+            _connectionGroupName = RuntimeHelpers.GetHashCode(this).ToString(NumberFormatInfo.InvariantInfo);
+
+            // Set HWR default values
+            _allowAutoRedirect = true;
+            _maxRequestContentBufferSize = HttpContent.MaxBufferSize;
+            _automaticDecompression = DecompressionMethods.None;
+            _cookieContainer = new CookieContainer(); // default container used for dealing with auto-cookies.
+            _credentials = null;
+            _maxAutomaticRedirections = 50;
+            _preAuthenticate = false;
+            _proxy = null;
+            _useProxy = true;
+            _useCookies = true; // deal with cookies by default.
+            _useDefaultCredentials = false;
+            _clientCertOptions = ClientCertificateOption.Manual;
```

Comment:
We have a file of defaults.  Should we be using that consistently to initialize these across platforms?